l5kit/l5kit/sampling/agent_sampling.py

```diff
@@ -275,6 +282,8 @@ def generate_agent_sample(
         "target_positions": future_positions_m,
         "target_yaws": future_yaws_rad,
         "target_velocities": future_vels_mps,
+        "target_speeds": future_speeds_mps,
```

Comment:
so this will be the new target for speed right?

Reply:
correct

Reply:
by the way, it is of shape [num_future_frames] and can be used to train model with longer time horizon.
I created another `"target_speed": future_speeds_mps[:1]` just for not breaking our existing model trained with single speed

Reply:
yeah I noticed that, good point